tests/approaches/test_oracle_approach.py

```diff
@@ -347,6 +348,32 @@ def test_oracle_approach_cover_multistep_options():
         assert policy_solves_task(policy, task, env.simulate)
 
 
+@longrun
```

Comment:
should we go through and mark some of our other tests as longrun? if we aren't feeling bottlenecked right now though, then it's okay to leave as is... here's this for reference on master:
```
6.03s call     tests/approaches/test_interactive_approach.py::test_interactive_learning_approach
5.31s call     tests/approaches/test_nsrt_learning_approach.py::test_grammar_search_invention_approach
4.45s call     tests/test_main.py::test_main
4.37s call     tests/test_predicate_search_score_functions.py::test_relaxation_energy_score_function
3.95s call     tests/test_predicate_search_score_functions.py::test_prediction_error_score_function
3.02s call     tests/test_predicate_search_score_functions.py::test_exact_energy_score_function
2.13s call     tests/approaches/test_nsrt_learning_approach.py::test_oracle_samplers
1.87s call     tests/nsrt_learning/test_nsrt_learning_main.py::test_nsrt_learning_specific_nsrts
1.76s call     tests/nsrt_learning/test_option_learning.py::test_oracle_option_learner_blocks
1.63s call     tests/test_predicate_search_score_functions.py::test_expected_nodes_score_function
1.63s call     tests/approaches/test_oracle_approach.py::test_oracle_approach_cover_multistep_options
1.43s call     tests/approaches/test_oracle_approach.py::test_oracle_approach_playroom
1.35s call     tests/approaches/test_oracle_approach.py::test_oracle_approach_painting
1.22s call     tests/approaches/test_nsrt_learning_approach.py::test_nsrt_learning_approach
1.14s call     tests/test_planning.py::test_sesame_plan_failures
1.12s call     tests/test_predicate_search_score_functions.py::test_count_score_functions
0.93s call     tests/envs/test_playroom.py::test_playroom_action_sequence_video
0.81s call     tests/test_models.py::test_mlp_classifier
0.77s call     tests/envs/test_base_env.py::test_env_creation
0.77s call     tests/test_predicate_search_score_functions.py::test_branching_factor_score_function
0.68s call     tests/approaches/test_nsrt_learning_approach.py::test_neural_option_learning
0.66s call     tests/test_predicate_search_score_functions.py::test_hadd_match_score_function
0.65s call     tests/test_predicate_search_score_functions.py::test_task_planning_score_function
0.58s call     tests/nsrt_learning/test_option_learning.py::test_known_options_option_learner
0.57s call     tests/nsrt_learning/test_option_learning.py::test_oracle_option_learner_cover
0.52s call     tests/approaches/test_nsrt_learning_approach.py::test_sampler_learning_with_goals
0.51s call     tests/approaches/test_oracle_approach.py::test_oracle_approach_cover
0.50s call     tests/approaches/test_oracle_approach.py::test_oracle_approach_cover_multistep_options_fixed_tasks
```

Reply:
presumably all or most of the longest ones are needed for coverage, otherwise we would've removed them already right?